plugins/woocommerce-blocks/docs/internal-developers/templates/block-template-controller.md

```diff
@@ -2,13 +2,13 @@
 
 ## Table of Contents <!-- omit in toc -->
 
--   [Overview](#overview)
--   [add_block_templates( $query_result, $query, \$template_type )](#add_block_templates-query_result-query-template_type-)
-    -   [Return value](#return-value)
--   [get_block_file_template( $template, $id, \$template_type )](#get_block_file_template-template-id-template_type-)
-    -   [Return value](#return-value-1)
--   [render_block_template()](#render_block_template)
-    -   [Return value](#return-value-2)
+- [Overview](#overview)
+- [add_block_templates](#add_block_templates)
+    - [add_block_templates return value](#add_block_templates-return-value)
+- [get_block_file_template](#get_block_file_template)
+    - [get_block_file_template return value](#get_block_file_template-return-value)
+- [render_block_template](#render_block_template)
+    - [render_block_template return value](#render_block_template-return-value)
```

Comment:
For some reason, markdownlint didn't like having function parameters as part of the titles. :confused: